src/NuGet.Core/NuGet.Build.Tasks.Pack/PackTaskLogic.cs

```diff
@@ -447,6 +447,19 @@ private IEnumerable<OutputLibFile> InitLibFiles(IMSBuildItem[] libFiles)
                             string.Format(CultureInfo.CurrentCulture, Strings.InvalidPlatformVersion, targetFramework)
                         );
                     }
+                    var isNet5EraTfm = fw.Version.Major >= 5 &&
+                        StringComparer.OrdinalIgnoreCase.Equals(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, fw.Framework);
+                    var isNetStandard = StringComparer.OrdinalIgnoreCase.Equals(FrameworkConstants.FrameworkIdentifiers.NetStandard, fw.Framework);
```

Comment:
```suggestion
                    var isNetStandard = !isNet5EraTfm && StringComparer.OrdinalIgnoreCase.Equals(FrameworkConstants.FrameworkIdentifiers.NetStandard, fw.Framework);
```
As per my understanding tfm cannot be both hence if tfm is already `net5.0` then we skip checking for `netstandard`